torch/onnx/utils.py

```diff
@@ -1703,38 +1703,38 @@ def _run_symbolic_function(
     """
 
     opset_version = GLOBALS.export_onnx_opset_version
-    symbolic_helper.is_caffe2_aten_fallback = symbolic_helper.is_caffe2_aten_fallback
 
     # See Note [Export inplace]
-    # TODO(ezyang): I think this is not necessary anymore
-    if n.kind().endswith("_"):
-        ns_op_name = n.kind()[:-1]
+    node_kind = n.kind()
+    if node_kind.endswith("_"):
+        # Treat relu_ -> relu; add_ -> add etc.
+        ns_op_name = node_kind[:-1]
     else:
-        ns_op_name = n.kind()
-    ns, op_name = ns_op_name.split("::")
+        ns_op_name = node_kind
+
+    namespace, op_name = ns_op_name.split("::")
 
     try:
         symbolic_registry.register_version("", opset_version)
 
         # Caffe2-specific: Quantized op symbolics are registered for opset 9 only.
         if symbolic_helper.is_caffe2_aten_fallback() and opset_version == 9:
-
             symbolic_caffe2.register_quantized_ops("caffe2", opset_version)
 
-        if ns == "aten":
+        if namespace == "aten":
```

Comment:
nit: Some of these are not related to "Remove unused patching methods"...

Reply:
I will break them out

Reply:
Actually I will keep them in if you are ok with it, since this function is touched by the removal (https://github.com/pytorch/pytorch/pull/83006/files/b305518f3ed8d57b78dcfcaeb861c6c53c2a5df2#diff-849a5778e2dcf7f36587967273cee0bf20642e35bf4c79405111ea3417c3fb3cR1748-R1751 etc) and a mild clean up doesn't seem too noisy?

Reply:
Mark as resolved for this one. I will break changes like this out in the future. Thanks!